File: src/test/console/java/src/main/java/util/DefaultCapabilities.java
package util;

import org.openqa.selenium.remote.DesiredCapabilities;

public class DefaultCapabilities {
  public static DesiredCapabilities getDefaultCapabilities(String platFormName) {
    DesiredCapabilities capabilities = new DesiredCapabilities();
    capabilities.setCapability("sessionName", "Android: java-jsonwired");
    capabilities.setCapability("sessionDescription", "Java Selenium Webdriver");
    capabilities.setCapability("deviceOrientation", "portrait");
    capabilities.setCapability("captureScreenshots", true);
    capabilities.setCapability("deviceGroup", "KOBITON");

    if (platFormName.equals("iOS")) {
      capabilities.setCapability("browserName", "safari");
      capabilities.setCapability("deviceName", "iPhone 5");
      capabilities.setCapability("platformName", "iOS");
    }
    else {

      capabilities.setCapability("browserName", "chrome");
      capabilities.setCapability("deviceName", "Galaxy J3");
      capabilities.setCapability("platformName", "Android");
    }
    return  capabilities;
  }
}

File: src/test/console/dotnet/Jsonwired/util/DefaultCapabilities.cs
using System;
using OpenQA.Selenium.Remote;

namespace util {
	public class DefaultCapabilities {
		public static DesiredCapabilities getDefaultCapabilities(string platFormName) {

			DesiredCapabilities capabilities = new DesiredCapabilities();
			capabilities.SetCapability("sessionDescription", "C# Selenium Webdriver");
			capabilities.SetCapability("deviceOrientation", "portrait");
			capabilities.SetCapability("deviceGroup", "KOBITON");
			capabilities.SetCapability("captureScreenshots", true);
			if (platFormName.Equals("iOS")) {
				capabilities.SetCapability("sessionName", "iOS: c#-jsonwired");
				capabilities.SetCapability("browserName", "safari");
				capabilities.SetCapability("deviceName", "iPhone 5");
				capabilities.SetCapability("platformName", "iOS");
			}
			else {
				capabilities.SetCapability("sessionName", "Android: c#-jsonwired");
				capabilities.SetCapability("browserName", "chrome");
				capabilities.SetCapability("deviceName", "Galaxy J3");
				capabilities.SetCapability("platformName", "Android");
			}
			return capabilities;
		}
	}
}